theseus/embodied/collision/tests/test_collision_factor.py

```diff
@@ -76,13 +76,18 @@ def test_collision2d_copy():
     assert cost_function2.name == "new_name"
 
 
-def test_collision2d_jacobians():
+@pytest.mark.parametrize("pose_cls", [th.Vector, th.SE2])
```

Comment:
Why not make th.Vector -> th.Point2 directly?

Reply:
Ah, sorry, when I first wrote this I wasn't sure if there was a `randn` method for `th.Point2`. I'll fix.

Reply:
Fixed.

Reply:
No worries. Thanks for the fix.